tests/testthat/indention_operators/eq_formals_complex_tokens-out.R

```diff
@@ -0,0 +1,30 @@
+function(a =
+           33,
+         b) {}
+
+function(a =
+           33,
+         b) {}
+
+function(a,
+         b,
+         c) {}
+
+function(a,
+         b,
+         c) {}
+
+function(ss,
+         a =
+           3,
+         er =
+           4) {}
+
+function(a =
+           b,
+         f =
+           d, c =
```

Comment:
Add newline after `,` if the argument consumes multiple lines?

Reply:
Multiline formals aren't covered either, because they're very rare. I think if a formal is spread over multiple lines, no other information should be contained on these lines.

Reply:
Deal 😊

Reply:
To do this properly, we actually need #357 because the `multi-line` attribute is not set just looking for line breaks at the current nest is insufficient.

Reply:
As this is part of milestone styler 1.0.1 (which was due a few days ago), I don't think I can implement #357 reasonably quickly. Should we do it improperly and add a FIXME or leave it as is and add a FIXME?

Reply:
Can we migrate this to a new issue? Because a merge of #353 would close #352 and #351 and all the other issues raised in #353 seem to be gone or declared as negligible.